binder/postBuild

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+# compile
+cd grass
+./configure \
+    --enable-largefile=yes \
+    --with-nls \
+    --with-cxx \
+    --with-readline \
+    --with-bzlib \
+    --with-pthread \
+    --with-proj-share=/usr/share/proj \
+    --with-geos=/usr/bin/geos-config \
+    --with-cairo \
+    --with-opengl-libs=/usr/include/GL \
+    --with-freetype=yes --with-freetype-includes="/usr/include/freetype2/" \
+    --with-sqlite=yes
+make
+
+# put command on path
+# ensure the user specific bin dir exists (already on path)
+mkdir -p $HOME/.local/bin/
+# create link to build
+ln -s $HOME/code/grass/bin.*/grass* $HOME/.local/bin/grass
```

Comment:
```suggestion
ln -s $HOME/grass/bin.*/grass* $HOME/.local/bin/grass
```

Reply:
@chaedri to test this use `!grass --version` in the notebook or `grass --version` in a shell you get in the JupyterLab.

Reply:
The first `grass` part is still one dir extra here even in the suggestion.